minikube/minikube.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash 
+set -x
+set -eu
+set -o pipefail
+
+up_registry () {
+  docker run -d -p 5000:5000 --restart=always --name registry registry:2
+}
+
+deploy_es () {
+  mkdir -p kube_es
+  pushd kube_es
+  for YAML in service-account.yaml es-svc.yaml es-rc.yaml ; do
+    wget https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/elasticsearch/$YAML
+    kubectl create -f $YAML
+  done
+  popd
+
+  while true; do
+    STATUS=$(kubectl get po | grep es | awk -F' ' '{print $3}')
+    if [ "$STATUS" == 'Running' ]; then
+      break
+    fi
+  done
+  ES_PORT=$(kubectl get svc elasticsearch -o json | jq -r '.spec.ports[] | select(.name == "http") | .nodePort')
+  MINIKUBE_IP=$(minikube ip)
+  echo "============================================"
+  echo "ElasticSearch adress:  $MINIKUBE_IP:$ES_PORT"
+  echo "============================================"
+}
+
+
+generate_config () {
+  CA_CERT=$(grep certificate-authority ~/.kube/config | awk -F" " '{print $2}')
+  CERT_FILE=$(grep client-certificate ~/.kube/config | awk -F" " '{print $2}')
+  KEY_FILE=$(grep client-key ~/.kube/config | awk -F" " '{print $2}')
+  REGISTRY_IP=$(hostname -i)
+  cat > ~/.ccp.yaml <<EOF
+builder:
+  push: True
+registry:
+  insecure: True
+  address: "$REGISTRY_IP:5000"
+nodes:
+  minikube:
+    roles:
+      - infra
+      - frontend
+      - backend
+kubernetes:
+  namespace: default
```

Comment:
Please, use the `oss` environment, also, to configure `kubectl` to use this environment add the following commands in the scripts:

```
kubectl config set-context oss --namespace oss
kubectl config use-context oss
```